src/test/java/spoon/test/constructor/ConstructorCallTest.java

```diff
@@ -0,0 +1,66 @@
+/**
+ * Copyright (C) 2006-2021 INRIA and contributors
+ * Spoon - http://spoon.gforge.inria.fr/
+ *
+ * This software is governed by the CeCILL-C License under French law and
+ * abiding by the rules of distribution of free software. You can use, modify
+ * and/or redistribute the software under the terms of the CeCILL-C license as
+ * circulated by CEA, CNRS and INRIA at http://www.cecill.info.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE. See the CeCILL-C License for more details.
+ *
+ * The fact that you are presently reading this means that you have had
+ * knowledge of the CeCILL-C license and that you accept its terms.
+ */
+package spoon.test.constructor;
+
+import org.junit.jupiter.api.Test;
+import spoon.Launcher;
+import spoon.reflect.code.CtConstructorCall;
+import spoon.reflect.code.CtExpression;
+import spoon.reflect.factory.Factory;
+import spoon.reflect.reference.CtTypeReference;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+public class ConstructorCallTest {
+
+    @Test
+    void testRemoveArgument() {
+        // contract: removeArgument removes the passed argument among two arguments in the call
+
+        Factory factory = new Launcher().getFactory();
+        CtExpression<?> argumentToBeRemoved = factory.Code().createCodeSnippetExpression("true").compile();
+        CtExpression<?> additionalArgument = factory.Code().createCodeSnippetExpression("false").compile();
+        CtConstructorCall<?> call = factory.Code().createConstructorCall(
```

Comment:
This code is not wrong in any way, but I wanted you to know that most of these methods are available on the core `Factory` object, and you don't need to access the actual subfactories.

```suggestion
        CtConstructorCall<?> call = factory.createConstructorCall(
```

Mentioning this as I've seen this a couple of times now in your recent PRs (where I've not bothered mentioning it because it's not a big deal).